ttnn/visualizer/app.py

```diff
@@ -243,6 +244,30 @@ def operations_with_l1_buffer_report():
     )
 
 
+@app.route("/perf_trace_report")
+def perf_trace_report():
+    report_path = get_report_path()
```

Comment:
https://github.com/tenstorrent/tt-metal/blob/47aafa7103715bceb71b9e05121ad6aa12e7ac45/tt_metal/tools/profiler/common.py#L25

Will give you the parent folder of the csv report. I would suggest we use it here so we don't fix the location to `ttnn.CONFIG.report_path / "tracy"`